src/to-svg.js

```diff
@@ -64,11 +64,13 @@ function addDefaultClassNames(classNames, key) {
  * @param {Object} options
  * @returns {string}
  */
-function optionsToAtrributes(options) {
+function optionsToAttributes(options) {
   const attributes = [];
 
   Object.keys(options).forEach(key => {
-    attributes.push(`${key}="${options[key]}"`);
+    if (options[key]) {
```

Comment:
When would `options[key]` ever be falsey in this context? `Object.keys(options)` is generating an array of all the keys in the object and we're using `forEach` to iterate over all the keys. I'm not sure `options[key]` would ever be undefined.

Reply:
In this case it was the lack of a value for `id` where `element.getAttribute('id')` results in `null`. This doesn't cause any errors but ends up with an eventual `id="null"` attribute on the SVG element. The only other situation where I could see this being falsey is if someone was to pass a `null` value when using `replace` or `toSvg`.

I could instead check if id has a value before passing it as an option to `toSvg`, let me know if that would be preferable here.

Reply:
Gotcha. I didn't think about that. I think this is fine!

Reply:
It may be a good idea to use TypeScript to catch these kinds of bugs 😄